workflow/controller/controller.go

```diff
@@ -320,6 +320,12 @@ func (wfc *WorkflowController) startLeading(ctx context.Context, logCtx *log.Ent
 	for i := 0; i < podWorkers; i++ {
 		go wait.Until(wfc.podWorker, time.Second, ctx.Done())
 	}
+	cacheGCPeriod := env.LookupEnvDurationOr("CACHE_GC_PERIOD", 0)
```

Comment:
global and log in init() too?

Reply:
should say, it is useful for users to be able to check their config

Reply:
Oh I missed this comment. Just updated. It might be a good idea to log all env vars at the start of the controller (in a potential follow-up PR).